wrappers/python/simtk/openmm/app/gromacstopfile.py

```diff
@@ -840,18 +882,25 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*unit.nan
 
                 for fields in moleculeType.atoms:
                     params = self._atomTypes[fields[1]]
-                    
+
                     if len(fields) > 6:
                         q = float(fields[6])
                     else:
                         q = float(params[4])
 
-                    if self._defaults[1] == '1':
+                    if has_nbfix_terms:
+                        if self._defaults[1] != '2':
+                            raise NotImplemented('NBFIX terms with LB combination rule is not yet supported')
```

Comment:
Don't raise `NotImplemented` -- it's for returning from operator overloads when they're not implemented.

```python
def __eq__(self, other):
    if not hasattr(other, 'some_attribute'):
        return NotImplemented
    return self.some_attribute == other.some_attribute
```

Raise `NotImplementedError()` instead.